framework/components/ecl-page-headers/ecl-page-headers.twig

```diff
@@ -36,40 +36,36 @@
 </div>
 {% else %}
 <div class="ecl-page-header">
-  {% include '@ec-europa/ecl-breadcrumbs' with { 'extraClass':'ecl-container','segments': [{'target':'#','title':'European Commission'},{'target':'#','title':'Announcements'}] } %}
-
-  <div class="ecl-page-header__body ecl-container">
-
-    {% if identity is defined %}
-    <div class="ecl-page-header__identity">
-      {{ identity }}
-    </div>
-    {% endif %}
-
-    {% if meta is defined %}
-    <div class="ecl-page-header__meta">
-      <div class="ecl-meta ecl-meta--header">
-        <span class="ecl-meta__item ecl-u-f-up">{{ meta.type }}</span>
-        <span class="ecl-meta__item">
-          <span class="date-display-single" property="dc:date" datatype="xsd:dateTime" content="{{ meta.timestamp }}">{{ meta.date }}</span>
-        </span>
-        <span class="ecl-meta__item">{{ meta.location }}</span>
+  <div class="ecl-container">
+    {% include '@ec-europa/ecl-breadcrumbs' with { 'segments': [{ 'target':'#', 'title':'European Commission' } , { 'target': '#', 'title': 'Announcements' }] } %}
```

Comment:
Does it make sense to move the
```
{ 'segments': [{ 'target':'#', 'title':'European Commission' } , { 'target': '#', 'title': 'Announcements' }] }
```
part in the context so that it can be overriden?